### ISSUE TITLE
Improve lint output

### ISSUE THREAD

**Issue by iamstarkov** [csscomb/csscomb.js#135]
CSScomb now have code style transformer, it's 

For JavaScript I use [node-jscs](https://github.com/mdevils/node-jscs) for linting js files. Is it possible to create css-linting tool for checking css on proper styleguides.

Is it possible to create this linting tool based on code from csscomjs?

**Comment by iamstarkov** [comment]
any news?

**Comment by tonyganch** [comment]
Blocked by #245, work in progress:
https://github.com/csscomb/csscomb.js/issues/245#issuecomment-106234425

**Comment by iamstarkov** [comment]
okie, so is this issue duplicate of #245?

**Comment by iamstarkov** [comment]
thanks for quick response

**Comment by tonyganch** [comment]
@iamstarkov, yes, I guess so
